src/types.ts

```diff
@@ -16,7 +16,7 @@ export type Point = {
 }
 
 export type Area = {
-  width: number
+  [width: string]: number
```

Comment:
why are you changing this? 🤔

Reply:
I'm sorry if it is a mistake, I'm not too familiar with Typescript.

I was getting a warning due to this [`this.state.croppedArea[attribute]`](https://github.com/ricardo-ch/react-easy-crop/pull/188/files#diff-64c7378dfabef2bf61425c20979c3762672c25a7ccd9a90ddef13222476f25a8R139) property access. It seemed like Typescript wanted me to explicitly declare that the keys within the `Area` object were `string` typed. Would this be the best way to go with it?

Reply:
I actually don't get this TS error when reverting this change. However I see what TS could complain, you can rewrite the `map()` line like this:
```ts
(['x', 'y', 'width', 'height'] as const).map((attribute) => {
...
})
```

Reply:
@ValentinH Thanks for help, this worked perfectly, I've reverted this change back to the original state.